baybe/acquisition/base.py

```diff
@@ -45,9 +45,25 @@ def to_botorch(
         """Create the botorch-ready representation of the function."""
         import botorch.acquisition as botorch_acqf_module
 
+        from baybe.acquisition.acqfs import qNegIntegratedPosteriorVariance
+
+        # Retrieve corresponding botorch class
         acqf_cls = getattr(botorch_acqf_module, self.__class__.__name__)
-        params_dict = filter_attributes(object=self, callable_=acqf_cls.__init__)
 
+        # Match relevant attributes
+        flds = fields(qNegIntegratedPosteriorVariance)
+        ignore = (
+            (
+                flds.sampling_n_points.name,
+                flds.sampling_method.name,
+                flds.sampling_fraction.name,
+            )
+            if isinstance(self, qNegIntegratedPosteriorVariance)
+            else ()
+        )
```

Comment:
Also not super happy yet, we can try to find a better solution. Reason why some form of "exception" is required: I think if we continue using an automated matching approach like the one we currently have in place, there should be a mechanism to ensure that the fields are actually used. This is what I changed in the current PR, because the bug the PR fixed was of the same kind – stuff that you expect to be passed got silently dropped, which is clearly terrible. I could also simply set `strict=False` to get it work, but that would defeat the purpose. So **somewhere** we need to specify which attributes go directly to botorch and which are used in a different way. Since the same functino also does the entire rest of "hardcoded" stuff (i.e. conversion to `mc_points` and so on), I thought this is probably the best place... 🤔 Happy to hear your suggestions

Reply:
if you want to be strict and add exceptions thats OK, but coding it here is a bad idea, better to code it in the class itself and here just access that ignore list

Reply:
How about the current version?

Reply:
looks fine, question about why its a property: I assume this is needed to access the attribute names? Thats not possible for a normal tuple class var I assume?